backend/copr_backend/sshcmd.py

```diff
@@ -243,8 +244,23 @@ def _rsync_download(self, src, dest, logfile, subprocess_timeout):
         log_filepath = "/dev/null"
         if logfile:
             log_filepath = os.path.join(dest, logfile)
-        command = "/usr/bin/rsync -rltDvH --chmod=D755,F644 -e '{}' {} {}/ &> {}".format(
-            ssh_opts, full_source_path, dest, log_filepath)
+
+        command = [
+            "/usr/bin/rsync",
+            "-rltDvH",
+            "--chmod=D755,F644",
+            "-e", "'{}'".format(ssh_opts),
+        ]
+        for value in include or []:
+            command.extend(["--include", shlex.quote(value)])
+        for value in exclude or []:
+            command.extend(["--exclude", shlex.quote(value)])
```

Comment:
Hm, I'm not sure I like the `include` and `exclude` split in this method.  The semantics of these options in rsync is order dependant;  it would be nicer if we had just one argument named `filter`  to have more flexibility for the future.